### ISSUE TITLE
I want to use Selectable node only how I can achieve this?

### ISSUE THREAD

**Issue by CrazyyCoder** [bmelnychuk/AndroidTreeView#54]
I'm new to Android and want to use this in my project as a tree view.
I want to implement it in MainActivity.
I have added dependency too but didn't understand how to use it.
Could you please help.

**Comment by bmelnychuk** [comment]
Did you look at sample project?

**Comment by CrazyyCoder** [comment]
yes I have test that but I'm not able to add Library in my project

On Wed, Feb 17, 2016 at 3:59 PM, Bogdan Melnychuk <notifications@github.com>
wrote:

> Did you look at sample project?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bmelnychuk/AndroidTreeView/issues/54#issuecomment-185142450>
> .
>



-- 
Best Regards

*Prakhar Chourasia*
Jr. Android Developer

*TBI Technologies*

**Comment by bmelnychuk** [comment]
Do you have gradle project? If so you can just add dependency to your build.gradle 
```compile 'com.github.bmelnychuk:atv:1.2.+'```
